docs/faq.rst

```diff
@@ -420,6 +421,36 @@ to change them.
 Security
 ########
 
+.. _borg_security_critique:
+
+Isn't BorgBackup's AES-CTR crypto broken?
+-----------------------------------------
+
+To exploit the AES counter management issue, an attacker would need to have
+access to the borg repository.
+
+By tampering with the repo, the attacker could bring the repo into a state so
+that it reports a lower "highest used counter value" than the one that actually
+was used. The client would usually notice that, because it rather trusts the
+clientside stored "highest used counter value" than trusting the server.
+
+But there are situations, where this is simply not possible:
+
+- If clients A and B used the repo, the client A can only know its own highest
+  CTR value, but not the one produced by B. That is only known to (B and) the
+  server (the repo) and thus the client A needs to trust the server about the
+  value produced by B in that situation. You can't do much about this except
+  not having multiple clients per repo.
+
+- Even if there is only one client, if client-side information is completely
+  lost (e.g. due to disk defect), the client also needs to trust the value from
+  server side. You can avoid this by not continuing to write to the repository
+  after you have lost clientside borg information.
+
+If a nonce (counter) value is reused, AES-CTR mode crypto is broken. This can
+be avoided by not using AES-CTR mode, but this needs bigger changes and won't
+happen soon.
```

Comment:
"won't happen soon" is relative to now and based on some assumptions.

first will change any day, second might not be true, so i'ld rather leave that away.

Reply:
ok